lib/Update.py

```diff
@@ -464,6 +465,7 @@ def ListClones():
                     ds.properties["usedbyrefreservation"].parsed + \
                     ds.properties["usedbysnapshots"].parsed
                 origin = ds.properties["origin"].parsed
+                tdict['dataset_space'] = ds.properties['used'].parsed
```

Comment:
Thats pretty much the same as rawspace, no?

Per man zfs: used = `usedbychildren + usedbydataset + usedbyrefreservation + usedbysnapshots`

Reply:
It changes here
```
                if '@' in origin:
                    snapshot = zfs.get_snapshot(origin)
                    tdict["rawspace"] += snapshot.properties["used"].parsed
```